functions/temp/User.js.json

```diff
@@ -0,0 +1 @@
+{"data":{"propertyOrder":{},"properties":{"id":{"name":"id","reference":"scalar","primary":true,"type":"number"},"username":{"name":"username","reference":"scalar","getter":false,"setter":false,"type":"string"},"email":{"name":"email","reference":"scalar","getter":false,"setter":false,"type":"string"},"password":{"name":"password","reference":"scalar","getter":false,"setter":false,"type":"string"},"profile_pic":{"name":"profile_pic","reference":"scalar","getter":false,"setter":false,"type":"string","nullable":true},"role":{"name":"role","reference":"scalar","enum":true,"items":["none","viewer","player"],"type":"UserRole"}},"props":[{"name":"id","reference":"scalar","primary":true,"type":"number"},{"name":"username","reference":"scalar","getter":false,"setter":false,"type":"string"},{"name":"email","reference":"scalar","getter":false,"setter":false,"type":"string"},{"name":"password","reference":"scalar","getter":false,"setter":false,"type":"string"},{"name":"profile_pic","reference":"scalar","getter":false,"setter":false,"type":"string","nullable":true},{"name":"role","reference":"scalar","enum":true,"items":["none","viewer","player"],"type":"UserRole"}],"primaryKeys":["id"],"filters":{},"hooks":{},"indexes":[],"uniques":[{"properties":"username"},{"properties":"email"}],"checks":[],"concurrencyCheckKeys":{},"className":"User","path":"./lib/bdd/entities/User.js","name":"User","abstract":false,"constructorParams":["userData","role"],"toJsonParams":[],"useCache":true,"compositePK":false,"relations":[],"collection":"user"},"origin":"./lib/bdd/entities/User.js","hash":"8b8619981d1ec3e516fb270fd8b51859","version":"5.1.5"}
```

Comment:
What is this ? 
Si c'est du temp du coup ça devrait pas être présent nan ?

Reply:
Mais en fait vous avez pas fait de .gitignore ? xD 
Je le rajoute mdrr